### ISSUE TITLE
Shipyard wont update at all

### ISSUE THREAD

**Issue by tulextreme** [cmdrmcdonald/EliteDangerousDataProvider#253]
Hello! I'm an avid EDDI user and just love all it's amazing functions and scripts! one thing thought is buggin me, the shipyard function is not updating at all on my end! After several resets from my ED api token its gathering less and less ships, and when switched to a new one the others that where good lose it's parameters making VA say "your ship" instead the ship's name... Any way to fiz this? Is there a file i could edit with the names correctly since i dont plan on selling any ships at all and there fore having it "fixed" in EDDI?
Thank you for any response i might get! Best regards
Cassio!

**Comment by ReverendRhyme** [comment]
I am also having this issue... Shipyard has ships listed I no longer own, and wont update or let me name them.

**Comment by tulextreme** [comment]
Nice a 10 day thread and no answers given,,, guess EDDI indeed  went to the trash bin... Well life goes on

**Comment by BeBumble** [comment]
From what I remember deleting or renaming the shipmonitor.json (in AppData/Roaming/EDDI) and then starting Elite & getting into each of your ships will populate the shipyard in EDDI database. Also it picks up the names of the ships in game and can't be changed in EDDI

btw that folder maybe invisible in windows so you will have to make it visible

**Comment by mwerle** [comment]
Not being able to override ship names anymore is a right royal pita for any long-term explorers who haven't been able to bane their ships in-game yet,  and won't be able to for some time yet. 

Don't understand why local override was disabled, unfortunately the old release no longer works for me or I'd roll back.

**Comment by tulextreme** [comment]
Yeah i could edit my ship names manually, changing the simple "name" string, but as for new ships it wont add any...
{
  "currentshipid": 22,
  "shipyard": [
    {
      "LocalId": 36,
      "model": "Diamondback Explorer",
      "cargocapacity": 16,
      "value": 10558092,
      **"name": "ensidius",**
      "ident": "TU-08D",
      "phoneticname": null,
      "role": "Multi-purpose",
      "raw": 

I'll delete and back up this shipmonitor.json and see the results!

**Comment by ReverendRhyme** [comment]
Removing the shipmonitor.json hasn't helped. Re-installing hasn't helped. It refuses to update names of any new ships. Plus, I'm missing over half my ships in the json file or I'd just edit it manually.

**Comment by tulextreme** [comment]
Yeah its fu*ed up! Mine behave so strangely it looks like it's alive...

**Comment by Tkael** [comment]
Unfortunately the shipyard is pretty bugged at the moment. It is a known issue. Comments from the developer are here:
https://forums.frontier.co.uk/showthread.php/294579-EDDI-Windows-app-for-immersion-and-more?p=5726100&viewfull=1#post5726100

**Comment by Solandus** [comment]
Do you guys use EDDI with VoiceAttack or standalone? This seems to happen only when we use EDDI as a VoiceAttack addin, right?

**Comment by Tkael** [comment]
as far as I know, this issue is attributable to the fact that much of the information regarding inactive ships was removed from the companion api. VoiceAttack vs standalone shouldn't affect it, but if anyone can verify that it does then that might be an interesting thread to pull.

**Comment by Solandus** [comment]
Well but the information of your ship actual in use got sended. But the Shipmonitor isn't working with it. There are two possibilites: First the ship monitor doesn't get this event, or it gets and the data is overwritten by an old deprecated event. Unfortunatelity I doesn't get how the code works (not enough experience with c#), so i couldn't fix it by myself. ;)

**Comment by Tkael** [comment]
There probably are could be some enhancements to the code that could make this more robust. The primary developer has stepped away from the project for the time being, but there's a small team (less experienced & still learning C#) working to understand the code and resolve these issues. This is a known issue on the list.
